libs/partners/openai/tests/integration_tests/chat_models/test_base.py

```diff
@@ -1041,8 +1041,32 @@ def test_o1_stream_default_works() -> None:
     assert len(result) > 0
 
 
+@pytest.mark.parametrize("use_responses_api", [False, True])
+@pytest.mark.parametrize("use_max_completion_tokens", [True, False])
+def test_minimal_reasoning_effort(
+    use_max_completion_tokens: bool, use_responses_api: bool
+) -> None:
+    if use_max_completion_tokens:
+        kwargs: dict = {"max_completion_tokens": MAX_TOKEN_COUNT}
+    else:
+        kwargs = {"max_tokens": MAX_TOKEN_COUNT}
+    response = ChatOpenAI(
+        model="gpt-5",
+        reasoning_effort="minimal",
+        use_responses_api=use_responses_api,
```

Comment:
```suggestion
        output_version="responses/v1",
```
(shouldn't make a difference here but we should just use this everywhere for the responses API)